pkg/apis/clusteroperator/types.go

```diff
@@ -378,6 +385,19 @@ type MachineSetStatus struct {
 
 	// Conditions includes more detailed status of the MachineSet
 	Conditions []MachineSetCondition
+
+	// Installed is true if the software required for this machine set is installed
+	// and running.
+	Installed bool
```

Comment:
@staebler I added this boolean to indicate that the install playbook had been run successfully on the masters. It needs to be reconciled with whatever you're doing for the master controller.

Reply:
ack